src/core/Table/filters/FilterToggle.tsx

```diff
@@ -53,21 +55,18 @@ export const FilterToggle = <T extends Record<string, unknown>>(
           onClickOutside={close}
           appendTo={ownerDocument?.body}
         >
-          <div
-            className={cx('iui-filter', {
-              'iui-active': isVisible || column.filterValue,
-            })}
+          <IconButton
```

Comment:
After going through all stories, this is the only problem I've noticed so far:
![image](https://user-images.githubusercontent.com/9084735/121746286-a3cd8f00-cad3-11eb-9b98-ac433b5c2841.png)

Reply:
So `padding-right` might actually be needed 😀

Reply:
Fixed but now I notice another problem :/

![image](https://user-images.githubusercontent.com/9084735/121938622-09ab5800-cd1a-11eb-9d67-bdf36bf1fa02.png)

Reply:
Can you add `title={undefined}` to line 347? Because `react-table` does this:
![image](https://user-images.githubusercontent.com/36186912/122002244-2ac38580-cdba-11eb-8f79-dfd7afa81ba7.png)
About sort going outside, I don't know what to do 😕 I see we had this problem previously, we either can get width using JS and apply it to all that column's cells or we can leave this to handle by the user if he wants to set fixed width. I would go with the second one.